dev/cognitiveservices/data-plane/Language/common.json

```diff
@@ -320,6 +320,24 @@
         }
       }
     },
+    "RequestWarning": {
+      "type": "object",
+      "description": "Contains details of warnings encountered for the whole task/request.",
+      "required": [
+        "code",
+        "message"
+      ],
+      "properties": {
+        "code": {
+          "type": "string",
```

Comment:
Is it possible to include the known warning codes like in DocumentWarning? https://github.com/Azure/azure-rest-api-specs/blob/main/specification/cognitiveservices/data-plane/Language/stable/2022-05-01/analyzetext.json#L2537